src/main/webapp/css/main.css

```diff
@@ -20,4 +20,17 @@ button {
     overflow: auto;
     margin-left: 5%;
     width: 90%;
-}
\ No newline at end of file
+}
+#logout{
```

Comment:
Could you please add indent between 'logout' and '{' to improve readability.
Same action for 'go-my-profile' .